data/pc/common/features.json

```diff
@@ -357,11 +482,11 @@
       },
       {
         "value": 5,
-        "version": "1.9_major"
+        "versions": "1.9_major"
```

Comment:
seems wrong no ?

Reply:
yeah def wrong https://github.com/PrismarineJS/node-minecraft-data/blob/637b4673f120aa25d50344f5b5d06a302bf6e5da/lib/supportsFeature.js#L34

Reply:
yeah sorry I thought I had gotten rid of all singular version typos

Reply:
but then again I think it was already there

Reply:
No exactly my point is it's not a typo
Single version is supported

Reply:
oh I see

Reply:
Fixed it